### PR TITLE
Hydrostatic pressure implementations

### PR DESCRIPTION
**Pull request by salvalatorre**

### CODE REVIEW

File: applications/DEMApplication/custom_constitutive/DEM_KDEM_Mohr_Coulomb_CL.cpp [hunk 1]
```diff
@@ -75,12 +75,11 @@ namespace Kratos {
             const double min_stress = *std::min_element(principal_stresses.begin(), principal_stresses.end());
             const double function_value = (max_stress - min_stress) + (max_stress + min_stress) * sinphi - 2.0 * mohr_coulomb_c * cosphi;
 
-            if(function_value > 0) {
+            if (function_value > 0) {
                 failure_type = 4;
-            }
-
+                *element1->mBrokenSphere = *element2->mBrokenSphere = 1.0;
```
Comment (maceligueta) [inline applications/DEMApplication/custom_constitutive/DEM_KDEM_Mohr_Coulomb_CL.cpp:98]: Breaks parallelism. You shouldn't write into two elements.

**Review by maceligueta — CHANGES_REQUESTED** [review]
This PR could be based on utilities. Instead, deep parts of the DEM application were modified. Let's discuss this in a meeting to see if we can improve it.

**Comment by philbucher** [comment]
is this still active?

**Comment by salvalatorre** [comment]
I don't think so... There are one or two interesting features that we will try to recover even after deleting this branch should we need them...
Thanks!

**Pull request closed without merging**